pkg/standalone/common.go

```diff
@@ -6,15 +6,31 @@ import (
 	"runtime"
 )
 
-// getDefaultComponentsFolder returns the hidden .components folder created at init time
-func GetDefaultComponentsFolder() string {
-	const daprDirName = ".dapr"
-	const componentsDirName = "components"
-	daprDirPath := os.Getenv("HOME")
+const (
+	defaultDaprDirName       = ".dapr"
+	DefaultComponentsDirName = "components"
+	defaultConfigDirName     = "config"
```

Comment:
@yaron2 @youngbupark 

I was reading the docs and I saw that in general for Dapr, configuration file is located inside `components` folder. Here I am separating it out into a specific folder called `config`. Is that something you think we should do or should I continue to use the `components` folder itself for defining the default config file.

Reply:
The default config should imo be at the root of the default dir.

Reply:
> The default config should imo be at the root of the default dir.

+1 let's keep it simple :)